CPC-CHARTER.md

```diff
@@ -156,6 +156,10 @@ ensuring:
       substantial changes before final approval by CPC.
    1. Voting members are responsible for approving funds for budgets delegated
       to the project.
+   1. Voting members are responsible for approving new `collaboration spaces` as outlined in the OpenJS 
+   Collaboration Network process. 
```

Comment:
@mhdawson Should we link this "Collaboration Network Process"? Might be helpful for anyone reading to navigate...